modules/monitoring-example-remote-write-authentication-settings.adoc

```diff
@@ -167,10 +165,9 @@ data:
 <1> The authentication type of the request. The default value is `Bearer`.
 <2> The name of the `Secret` object that contains the authentication credentials.
 <3> The key that contains the authentication token in the specified `Secret` object.
-====
 
-.Sample YAML for OAuth 2.0 authentication
-====
+== Sample YAML for OAuth 2.0 authentication
```

Comment:
[All headings in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings) take an anchor ID.